Moved 'Azure - Scan for Anonymous Access to Azure Storage (Powershell)' from T1530 to T1619

manuelkuss opened the pull request:
Moved 'Azure - Scan for Anonymous Access to Azure Storage (Powershell)' from T1530 to T1619

**Details:**
By the description of the atomic test, it tests for anonymous access to Azure storage containers by invoking a web request. "The corresponding response could then be interpreted to determine whether or not the resource/container exists". T1619 (Cloud Storage Object Discovery) fits much better than T1530 (Data from Cloud Storage) for the objective of the atomic test. T1619 is a discovery technique, T1530 is a collection technique. Furthermore, the atomic tests only stores information on whether it was possible to access the storage object, or not. It doesn't collect any storage content.

patel-bhavin reviewed (changes requested):
minor change requested and please remove the conflicts!

manuelkuss commented:
Hi!
Thanks, and good suggestion! 
I changed the parameter names to match the new technique.
I changed also the name for one parameter name in the atomic "Azure - Enumerate Azure Blobs with MicroBurst", which i moved in a previous pull request.
